2-ui/99-ui-misc/01-mutation-observer/article.md

```diff
@@ -236,9 +236,9 @@ There's a method to stop observing the node:
 
 - `observer.disconnect()` -- stops the observation.
 
-When we stop the observing, it might be possible that some changes were not processed by the observer yet.
+When we stop the observing, it might be possible that some changes were not yet processed by the observer.
```

Comment:
```suggestion
When we stop the observing, it might be possible that some changes were not yet processed by the observer. In that case, we can call
```